### PR TITLE
create gis-wisdom blog post

### PR DESCRIPTION
**Pull request by gregbunce**
Pull request description
------------------------

### CODE REVIEW

**Comment by gregbunce** [comment]
hi, @atribolet  here's another post I was hoping you can take a look at.  no rush, i'm just opening up the conversation.  thanks!

File: _posts/2019-04-24-gis-wisdom.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+Likewise, building a personal network helps us determine who and what we allow ourselves to connect with.  A well-thought-out network not only tells us where we can connect but also where we should not. Taking the time to think about this connectivity ensures that we build a quality social network.
+
+Learn more about [creating an ArcGIS network datasets](http://desktop.arcgis.com/en/arcmap/latest/extensions/network-analyst/creating-a-network-dataset.htm) and [how connectivity works](http://desktop.arcgis.com/en/arcmap/latest/extensions/network-analyst/understanding-connectivity.htm).
+
+### 3. Extract, Translate, and Load (ETL) 
+{: .text-left}
+
+`being adaptable`
+
+You have probably heard the phrase, 'build once, use many'.  When building data, we say this when we want the output to be adaptable and used for many purposes.  Take road centerlines for example.  This dataset should be designed with many uses in mind (such as routing, mapping, locating, assessment management, etc.).  With a simple ‘extract, translate, and load’ (ETL), the data should meet the needs of many users.
```
Comment (steveoh) [inline _posts/2019-04-24-gis-wisdom.md]: `locating, assessment management, etc.).` did you mean assessment or possibly asset?

File: _posts/2019-04-24-gis-wisdom.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+But it's the refinement process that makes it valuable.  We too can continually refine through learning and sharing and practice and performance.  In turn, this advances our skills and adds to our value.
+
+Brush-up on some [essential editing vocabulary](http://desktop.arcgis.com/en/arcmap/10.3/manage-data/editing/essential-editing-vocabulary.htm).
+
+### 6. Domains
+{: .text-left}
+
+`consistency`
+
+Domains are set up in the database to establish known consistently. They provide quality assurance on the front-end while ensuring consistency on the back-end.  What makes domains so great is that the end-user can reliably expect certain values or results.  Developers and cartographers benefit from this as well.
```
Comment (steveoh) [inline _posts/2019-04-24-gis-wisdom.md]: `Domains are set up in the database to establish known consistently.` Is `consistently` the word you want here?

File: _posts/2019-04-24-gis-wisdom.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+Learn more about [attribute domains](https://pro.arcgis.com/en/pro-app/help/data/geodatabases/overview/an-overview-of-attribute-domains.htm) and why they are so important. 
+
+### 7. Topology
+{: .text-left}
+
+`establishing principles`
+
+Topology is all about rules.  Essentially, it allows us to keep our data clean by defining a set of spatial expectations.  We are able to validate when needed and also allow for the inevitable exception.  Topology is so powerful because once you define the rules, you are then free to focus your efforts on other aspects of the data.
+
+Similarly, we also benefit from predefined rules -- or better yet, principles.  For example, think about driving an automobile.  We learn the rules-of-the-road before set foot in the vehicle.  Why?  Because we don’t want to create the rules ‘on-the-fly’.  Knowing the rules-of-the-road in advance, allows us to focus on more enjoyable things, such as conversation or a good audiobook.
```
Comment (steveoh) [inline _posts/2019-04-24-gis-wisdom.md]: `We learn the rules-of-the-road before set foot in the vehicle.` this sounds funny. Is there a grammar issue?

File: _posts/2019-04-24-gis-wisdom.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+Brush-up on some [essential editing vocabulary](http://desktop.arcgis.com/en/arcmap/10.3/manage-data/editing/essential-editing-vocabulary.htm).
+
+### 6. Domains
+{: .text-left}
+
+`consistency`
+
+Domains are set up in the database to establish known consistently. They provide quality assurance on the front-end while ensuring consistency on the back-end.  What makes domains so great is that the end-user can reliably expect certain values or results.  Developers and cartographers benefit from this as well. 
+
+Much like with domains, consistency can lead to reliable and expected results in human performance as well.  Think of athletes or musicians.  It's their consistently (and consequently, the establishment of muscle memory) that make the best so reliable and successful. Sure, repetition plays a role, but without pre-established methods (ie: domains) the results vary.
```
Comment (nathankota) [inline _posts/2019-04-24-gis-wisdom.md]: One last consistently to consistency change here.

**Comment by stdavis** [comment]
This is really great! The only suggestion that I have is to stick your email address in at the end. You ask for emails but there's not an address provided.

File: _posts/2019-04-24-gis-wisdom.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+Topology is all about rules.  Essentially, it allows us to keep our data clean by defining a set of spatial expectations.  We are able to validate when needed and also allow for the inevitable exception.  Topology is so powerful because once you define the rules, you are then free to focus your efforts on other aspects of the data.
+
+Similarly, we also benefit from predefined rules -- or better yet, principles.  For example, think about driving an automobile.  We learn the rules-of-the-road before we get behind the wheel.  Why?  Because we don’t want to create the rules ‘on-the-fly’.  Knowing the rules-of-the-road in advance, allows us to focus on more enjoyable things, such as conversation or a good audiobook.
+
+Oddly, there's something freeing about having predetermined principles.  Knowing our principles allows us to then explore greater things.
+
+Learn the [basics of topology](https://pro.arcgis.com/en/pro-app/help/data/topologies/topology-basics.htm) and how it might assist you with data management.
+
+<br/>
+Drop me an [email](mailto:gbunce@utah.gov) If you can think of other forms of GIS wisdom.  I'll start a running list and post them below.
```
Comment (steveoh) [inline _posts/2019-04-24-gis-wisdom.md]: I don't like having emails directly in posts. I would prefer we use [data contacts](https://github.com/agrc/gis.utah.gov#contact-information). Would you switch to that. Let me know if you need help.

**Comment by atribolet** [comment]
Hey, everyone! I'm a bit booked for the next few days, but I'll get to this as soon as I can :) I'll put it on my to-do list. Keep you posted!

File: _posts/2019-04-24-gis-wisdom.md (hunk 5 above)
Comment (gregbunce) [reply _posts/2019-04-24-gis-wisdom.md]: yup, I knew that.  oops.

**Comment by gregbunce** [comment]
thanks for the heads-up, @atribolet

**Comment by gregbunce** [comment]
hi, @atribolet just checking in.  do you think you'll have time this week to review this?  thanks!

**Comment by atribolet** [comment]
Hey! I'm hoping I will! Sorry for the delay so far! I'm helping prepare documentation for the upcoming audit, and it's been a little crazy. But I'll see what I can do this week! I'm hoping to get the other documentation under control in the next couple days. :) Keep you posted!

**Comment by atribolet** [comment]
Maybe this is a silly suggestion, but would it be better to go ahead and post it (so you're not waiting on me so much), and then I can edit it as soon as I can? Just a thought!

**Comment by gregbunce** [comment]
ouch, sounds busy.  good luck!  yeah, I like your idea of us posting it and going from there.  @steveoh , what do you think?  I'm okay with that.  thanks, @atribolet

**Comment by steveoh** [comment]
merged

**Pull request closed without merging**